src/components/project/SortList.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
+
+import { SortAscending, SortDescending } from '@phosphor-icons/react/dist/ssr';
+import { useState } from 'react';
+
+import { Chapter, type ChapterProps } from './Chapter';
+import { Title } from '../common/Title';
+
+interface SortListProps {
+  title: string;
+  items: ChapterProps[];
+}
+
+export function SortList({ title, items }: SortListProps) {
+  const [sortedItems, setSortedItems] = useState<ChapterProps[]>(items);
+  const [isAscending, setIsAscending] = useState<boolean>(false);
+
+  const handleSorting = () => {
+    const sorted = [...sortedItems].sort((a, b) =>
+      isAscending
+        ? a.date.toISOString().localeCompare(b.date.toISOString())
+        : b.date.toISOString().localeCompare(a.date.toISOString()),
+    );
+    setSortedItems(sorted);
+    setIsAscending(!isAscending);
+  };
+
+  return (
+    <div>
+      <Title title={title} className="mb-12 pr-6">
+        <button onClick={handleSorting}>
```

Comment:
Adicionar propriedade "title" para indicar a função do botão.

obs: esse title será substituído por um popover futuramente.

Reply:
Na real deixei isso ai como botão temporariamente... estava lendo sobre umas convesões.. e gostei de uma ideia em que só se usa a tag buttom dentro de forms, e gostei kkkk. Então posteriormente devo tirar esse botão quando eu for refatorar todas os componentes.

Reply:
não sei bem o que tu vai usar no lugar do buttom, mas tudo bem, revisamos quando refatorar.